test/arguments.js

```diff
@@ -1,3 +1,4 @@
+import {URL} from 'whatwg-url';
```

Comment:
Sure. It'd be nice to test both the native and shim. I'm swapping it out. Just to make sure, how does the rest feel about going from a lib that's seen tons of use to something that's still in alpha?